superx/information_extractors/item_info_extractor.py

```diff
@@ -47,7 +47,13 @@ def run_info_extractor(self):
                     logging.error(str(ce))
 
             zip_links = self.get_zip_file_links(url_list)
-            self.extract_xml_from_zip_and_parse(zip_links)
+            node_info_list = self.extract_xml_from_zip_and_parse(zip_links)
+
+            for info_child_node, branch_id in node_info_list:
+                info_tuple_list = self.extract_information_from_parsed_xml(info_child_node)
+                db.session.commit()
+                self.fill_branch_price_table(info_tuple_list, branch_id)
+                db.session.commit()
```

Comment:
I used 2 commints for testing. from what I saw it is faster to do it this way, I am trying to play around with 1 commit and see how I can improve the speed